source/gameengine/Ketsji/KX_PythonInit.cpp

```diff
@@ -2544,7 +2544,7 @@ void createPythonConsole()
   BLI_path_append(filepath, sizeof(filepath), "interpreter.py");
 
   // Use _Py_fopen to make sure we use the same fopen function as python use.
```

Comment:
Maybe the comment should modify to "// Use fopen as _Py_fopen was removed from Python 3.10" or something like that.

Reply:
Oh, I missed it somehow. Updated the PR after removing the comment. Thanks for the review!